src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -337,6 +337,11 @@ private void ProcessExpressionParameter()
             if (ExcludeProperty != null)
             {
                 _exclusionFilter = new MshExpressionFilter(ExcludeProperty);
+                if ((Property == null) || (Property.Length == 0))
+                {
```

Comment:
I think you mean that it's very **un**intuitive 😕 

If the intention is (to maintain the current expected behavior) that `-ExpandProperty` ignores `-ExcludeProperty` wouldn't it be better to have these switches in separate parameter sets? I'd certainly rather have it be _obvious that they can't be used together_ than to allow them together but ignore one (or worse, produce runtime errors if they're used together)...